Bowling/NUnitTestBowling/BowlingTests.cs

```diff
@@ -0,0 +1,70 @@
+using NUnit.Framework;
+using System.Runtime.InteropServices;
+using BowlingApp;
+using System.IO;
+using System;
+using System.Collections.Generic;
+
+namespace NUnitTestBowling
+{
+    public class Tests
+    {
+
+        [Test]
+        [TestCase("XXXXXXXXXXXX", "300", Description = "Perfect Game")]
+        [TestCase("9-9-9-9-9-9-9-9-9-9-", "90", Description = "Open Frames with -")]
+        [TestCase("5/5/5/5/5/5/5/5/5/5/5", "150", Description = "All Spares")]
+        [TestCase("548/XX725/7/52X6/5", "158", Description = "Random Open Spares Strikes")]
+        [TestCase("54637281233445625381", "82", Description = "All Open")]
```

Comment:
Nice use of the `TestCaseAttribute`!

Reply:
Piggy backing off of Austin.

These are very good tests to have.  These would be considered integration tests.  I would like to see how we could write more unit tests for this kata.  The advantage of unit testing vs integration testing is when an integration test breaks, it's nice to know something went wrong, but it becomes hard to figure out where and why.  I think after your next round of refactoring it'll be more apparent how we can introduce unit tests.  It's hard (or impossible) to unit test code with larger classes and long functions.  When breaking code down into more class (single responsibilities) with short functions, unit testing becomes a lot easier!  This is not an easy skill to pick up so ask for help if you need it.

Reply:
Another thing:

These tests are verifying the happy path...but what should I expect to happen if test case had an attribute like the following?

`[TestCase("ABCDEFG", "???", Description = "The user messed up the input")]`

I also saw you wrote code that used a regex to parse that spaces were removed...that would be good to test too!